src/Control/Comonad/Cofree.purs

```diff
@@ -39,8 +39,8 @@ newtype Cofree f a = Cofree (Lazy (Tuple a (f (Cofree f a))))
 
 -- | Lazily creates a value of type `Cofree f a` from a label and a
 -- | functor-full of "subtrees".
-liftC :: forall f a. (Unit -> Tuple a (f (Cofree f a))) -> Cofree f a
-liftC = Cofree <<< defer
+deferCofree :: forall f a. (Unit -> Tuple a (f (Cofree f a))) -> Cofree f a
+deferCofree = Cofree <<< defer
```

Comment:
I've called this `deferCofree` since it's just a specialized implementation of the `Lazy` instance.